ext/mini_racer_extension/extconf.rb

```diff
@@ -19,7 +19,7 @@
 $CXXFLAGS += " -g" unless $CXXFLAGS.split.include? "-g"
 $CXXFLAGS += " -rdynamic" unless $CXXFLAGS.split.include? "-rdynamic"
 $CXXFLAGS += " -fPIC" unless $CXXFLAGS.split.include? "-rdynamic" or IS_DARWIN
-$CXXFLAGS += " -std=c++17"
+$CXXFLAGS += " -std=c++20"
```

Comment:
Do we need to update the README for compiler requirements too? https://github.com/rubyjs/mini_racer?tab=readme-ov-file#installation

Reply:
Good point, those are way out of date. Updated with the compiler recommendations from node's BUILDING.md